<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbScience">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { background-color: rgb(255, 255, 255);  border: 3px solid rgb(255, 0, 0); }</string>
    </property>
    <property name="title">
     <string>Wissenschaft</string>
    </property>
    <widget class="QPushButton" name="pbAstros">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>akt. Raumfahrer anzeigen</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Bilder/Icons/icons8-astronaut-64.png</normaloff>../../../Bilder/Icons/icons8-astronaut-64.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pbWetter">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>akt. Wetter</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Bilder/Icons/icons8-sonne-64.png</normaloff>../../../Bilder/Icons/icons8-sonne-64.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pbApod">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>APOD from NASA</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Bilder/Icons/observatory.png</normaloff>../../../Bilder/Icons/observatory.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pbEpic">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>EPIC(Earth Polychromatic Imaging Camera) from NASA</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Bilder/Icons/icons8-europa-96.png</normaloff>../../../Bilder/Icons/icons8-europa-96.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pbMarsRoverPictures">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Get Mars Rover Pictures</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Bilder/Icons/icons8-mars-60.png</normaloff>../../../Bilder/Icons/icons8-mars-60.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLandsat">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Get Landsat Images</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons8-satellite-im-orbit-80.png</normaloff>icons8-satellite-im-orbit-80.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>51</width>
       <height>51</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbEnde">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Beenden</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbDevelopment">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { background-color: rgb(255, 255, 255);  border: 3px solid rgb(255, 0, 0); }</string>
    </property>
    <property name="title">
     <string>Development</string>
    </property>
    <widget class="QPushButton" name="pbColors">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Convert Colors</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Bilder/Icons/icons8-colors-58.png</normaloff>../../../Bilder/Icons/icons8-colors-58.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbFun">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { background-color: rgb(255, 255, 255);  border: 3px solid rgb(255, 0, 0); }</string>
    </property>
    <property name="title">
     <string>Spa√ü</string>
    </property>
    <widget class="QPushButton" name="pbBored">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Things to do when bored</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Bilder/Icons/icons8-bmo-50.png</normaloff>../../../Bilder/Icons/icons8-bmo-50.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pbXKCD">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>XKCD Comic</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Bilder/Icons/television_klein.png</normaloff>../../../Bilder/Icons/television_klein.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLyrics">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Songtexte</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Bilder/Icons/icons8-gesangslehrer-80.png</normaloff>../../../Bilder/Icons/icons8-gesangslehrer-80.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pbRecipeDB">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Rezepte suchen</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Bilder/Icons/icons8-recipe-book-64.png</normaloff>../../../Bilder/Icons/icons8-recipe-book-64.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>51</width>
       <height>51</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuPublic_REST_APIs">
    <property name="title">
     <string>Public REST APIs</string>
    </property>
    <addaction name="actionXkcd"/>
    <addaction name="actionAstros"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="action_ber"/>
   </widget>
   <widget class="QMenu" name="menuDevelopment">
    <property name="title">
     <string>Development</string>
    </property>
    <addaction name="actionBored"/>
    <addaction name="actionColors"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuPublic_REST_APIs"/>
   <addaction name="menuDevelopment"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionXkcd">
   <property name="text">
    <string>Xkcd</string>
   </property>
  </action>
  <action name="actionAstros">
   <property name="text">
    <string>Astros</string>
   </property>
  </action>
  <action name="action_ber">
   <property name="text">
    <string>√úber</string>
   </property>
  </action>
  <action name="actionLyrics">
   <property name="text">
    <string>Lyrics</string>
   </property>
   <property name="toolTip">
    <string>Suche Songtexte</string>
   </property>
  </action>
  <action name="actionBored">
   <property name="text">
    <string>Bored</string>
   </property>
   <property name="toolTip">
    <string>Activities to do when Bored</string>
   </property>
  </action>
  <action name="actionColors">
   <property name="text">
    <string>Colors</string>
   </property>
   <property name="toolTip">
    <string>Convert Colorcodes</string>
   </property>
  </action>
  <action name="actionWetter">
   <property name="text">
    <string>Wetter</string>
   </property>
   <property name="toolTip">
    <string>akt. Wetter</string>
   </property>
  </action>
  <action name="actionGetApod">
   <property name="text">
    <string>getApod</string>
   </property>
   <property name="toolTip">
    <string>APOD from NASA</string>
   </property>
  </action>
  <action name="actionNasaEpic">
   <property name="text">
    <string>NasaEpic</string>
   </property>
   <property name="toolTip">
    <string>EPIC(Earth Polychromatic Imaging Camera) from NASA</string>
   </property>
  </action>
  <action name="actionMarsRoverPictures">
   <property name="text">
    <string>MarsRoverPictures</string>
   </property>
   <property name="toolTip">
    <string>Get Mars Rover Pictures</string>
   </property>
  </action>
  <action name="actionRecipeDB">
   <property name="text">
    <string>RecipeDB</string>
   </property>
   <property name="toolTip">
    <string>Rezepte suchen</string>
   </property>
  </action>
  <action name="actionLandsat">
   <property name="text">
    <string>Landsat</string>
   </property>
   <property name="toolTip">
    <string>Get Landsat Imagery</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbEnde</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbXKCD</sender>
   <signal>clicked()</signal>
   <receiver>actionXkcd</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBeenden</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAstros</sender>
   <signal>clicked()</signal>
   <receiver>actionAstros</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbLyrics</sender>
   <signal>clicked()</signal>
   <receiver>actionLyrics</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbBored</sender>
   <signal>clicked()</signal>
   <receiver>actionBored</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbColors</sender>
   <signal>clicked()</signal>
   <receiver>actionColors</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbWetter</sender>
   <signal>clicked()</signal>
   <receiver>actionWetter</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbApod</sender>
   <signal>clicked()</signal>
   <receiver>actionGetApod</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbEpic</sender>
   <signal>clicked()</signal>
   <receiver>actionNasaEpic</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbMarsRoverPictures</sender>
   <signal>clicked()</signal>
   <receiver>actionMarsRoverPictures</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbRecipeDB</sender>
   <signal>clicked()</signal>
   <receiver>actionRecipeDB</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbLandsat</sender>
   <signal>clicked()</signal>
   <receiver>actionLandsat</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
